docs/proposals/20220209-machinepool-machines.md

```diff
@@ -87,36 +94,95 @@ As another example, currently each MachinePool infrastructure provider must impl
 
 ## Proposal
 
-To create MachinePool Machines, a MachinePool in CAPI needs information about the instances or replicas associated with the provider's implementation of the MachinePool. This information is attached to the provider's MachinePool infrastructure resource in new status fields `InfrastructureMachineSelector` and `InfrastructureMachineKind`. These fields should be populated by the infrastructure provider.
+
+### Enabling MachinePoolMachines
+
+To enable the MachinePool Machines feature, a MachinePool in CAPI needs to know the type of the instances or replicas associated with the provider's implementation of the MachinePool. This information is attached to the provider's MachinePool infrastructure resource in the new status field `InfrastructureMachineKind`. When this field is populated by the provider, it signals to the Cluster API MachinePool controller that this InfrastructureMachinePool is opted-in to support MachinePool Machines.
 
 ```golang
-// FooMachinePoolStatus defines the observed state of FooMachinePool.
-type FooMachinePoolStatus struct {
-	// InfrastructureMachineSelector is a label query over the infrastructure resources behind MachinePool Machines.
-	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
-	// +optional
-	InfrastructureMachineSelector metav1.LabelSelector `json:"infrastructureMachineSelector,omitempty"`
+// InfraMachinePoolStatus defines the observed state of an InfraMachinePool.
+type InfraMachinePoolStatus struct {
 	// InfrastructureMachineKind is the kind of the infrastructure resources behind MachinePool Machines.
 	// +optional
 	InfrastructureMachineKind string `json:"infrastructureMachineKind,omitempty"`
 }
 ```
 
-These fields are an addition to the optional status fields of `InfrastructureMachinePool` in the [provider contract][].
+This field is in addition to the optional status fields of `InfrastructureMachinePool` in the provider contract.
```

Comment:
was removal of the link intentional?

link is defined at the bottom of the doc https://github.com/kubernetes-sigs/cluster-api/pull/9664/files#diff-b6daaed4829762aa7988f763c28252304b3ad374dc238863db6b182258c97f6dL215

Reply:
Oh I see, I didn't realize you could create links like that.